# Introdução à Ciência de Dados - UFPB
Professor: Yuri Malheiros

## Medidas de Centralidade e Dispersão

A partir do dataset `movies.csv` calcule a média, mediana e desvio padrão para as colunas:

In [ ]:
import pandas as pd

filmes = pd.read_csv('movies.csv')

### US Gross

In [ ]:
media_us_gross = filmes['US Gross'].mean()
mediana_us_gross = filmes['US Gross'].median()
std_us_gross = filmes['US Gross'].std()

print('Média: %f \t Mediana: %f \t Desvio padrão: %f '%(media_us_gross,mediana_us_gross,std_us_gross))

Média: 44002085.163745 	 Mediana: 22019465.500000 	 Desvio padrão: 62555311.390662 


### Production Budget

In [ ]:
media_production_budget = filmes['Production Budget'].mean()
mediana_production_budget = filmes['Production Budget'].median()
std_production_budget = filmes['Production Budget'].std()

print('Média: %f \t Mediana: %f \t Desvio padrão: %f '%(media_production_budget,mediana_production_budget,std_production_budget))

Média: 31069171.448438 	 Mediana: 20000000.000000 	 Desvio padrão: 35585913.444644 


### Running Time

In [ ]:
media_time = filmes['Running Time (min)'].mean()
mediana_time = filmes['Running Time (min)'].median()
std_time = filmes['Running Time (min)'].std()

print('Média: %f \t Mediana: %f \t Desvio padrão: %f '%(media_time,mediana_time,std_time))

Média: 110.193548 	 Mediana: 107.000000 	 Desvio padrão: 20.171014 


### IMDB Rating

In [ ]:
media_IMDB = filmes['IMDB Rating'].mean()
mediana_IMDB = filmes['IMDB Rating'].median()
std_IMDB = filmes['IMDB Rating'].std()

print('Média: %f \t Mediana: %f \t Desvio padrão: %f '%(media_IMDB,mediana_IMDB,std_IMDB))

Média: 6.283467 	 Mediana: 6.400000 	 Desvio padrão: 1.252290 


### IMDB Votes

In [ ]:
media_votes = filmes['IMDB Votes'].mean()
mediana_votes = filmes['IMDB Votes'].median()
std_votes = filmes['IMDB Votes'].std()

print('Média: %f \t Mediana: %f \t Desvio padrão: %f '%(media_votes,mediana_votes,std_votes))

Média: 29908.644578 	 Mediana: 15106.000000 	 Desvio padrão: 44937.582335 


### Análise dos Resultados

Quais colunas tiveram valores muito diferentes entre média e mediana? Quais os seus desvios padrão?

**As colunas US Gross, Production Budget e IMDB Votes com seus respectivos desvios padrões:**
<ul>
<li> <b>62555311.390662</b> </li>
<li> <b>35585913.444644</b> </li>
<li> <b>44937.582335</b> </li>
</ul>
 

Que conclusões podemos tirar desses resultados?

**Com esses desvios tão altos, podemos concluir que existem dados com valores enormes e/ou com valores pequenos, assim esses valores estão distantes da média, ou seja, bem espalhados.**